portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase

```diff
@@ -647,6 +647,62 @@ definition {
 			fieldValue = "Admin Comments");
 	}
 
+	@description = "This is a use case for LPS-119818."
+	@priority = "5"
+	test DeleteFieldThatBelongsToARule {
+		property portal.acceptance = "true";
```

Comment:
@alinedoleron, make sure Cleyton is comfortable with this information, ok? Please answer me here with his inputs. Thanks.

Reply:
He said he wouldn't put this `property portal.acceptance = "true";` because it's a new thing and we need monitore before putting it inside relevant command. So I'll remove this line.